macro/README.md

```diff
@@ -1,66 +1,22 @@
-# Iroha Macro
+# Iroha Macros
 
-## Description
+This crate contains macros and attributes for Iroha projects:
 
-This crate contains macros and attributes for Iroha projects.
-`iroha_derive` contains derive macros.
-
-### Features
-
-* `log` attribute for debugging functions inputs and output
+- `iroha_derive` is a custom derive macros for Iroha
```

Comment:
Well, not quite right. `iroha_derive` is just another crate (we have to use crate inside crate for this kind of macros, it's a Rust requirement).
I think, you wanted to describe available macros. The only macro available now is `FromVariant`. I've left a link to it in another comment

Reply:
fixed (I think)